src/react/src/components/Contribute/ProductionLocationInfo.jsx

```diff
@@ -0,0 +1,523 @@
+                                    onChange={setProcessingType}
+                                    styles={selectStyles}
+                                    className={classes.selectStyles}
+                                />
+                            </div>
+                            <div
+                                className={`${classes.inputSectionWrapStyles} ${classes.wrapStyles}`}
+                            >
+                                <Typography
+                                    component="h2"
+                                    className={classes.titleStyles}
+                                >
+                                    Number of Workers
+                                </Typography>
+                                <Typography
+                                    component="h4"
+                                    className={classes.subTitleStyles}
+                                >
+                                    Select a number or a range for the number of
+                                    people employed at the location. For
+                                    example: 100, 100-150.
+                                </Typography>
+                                <StyledSelect
+                                    id="number_of_workers"
+                                    name="Number of Workers"
+                                    options={numberOfWorkersOptions || []}
+                                    value={numberOfWorkers}
+                                    onChange={setNumberOfWorkers}
+                                    placeholder="Enter the number of workers as a number or range"
+                                    aria-label="Number of workers"
+                                    isMulti={false}
+                                    styles={selectStyles}
+                                    className={classes.selectStyles}
+                                />
+                            </div>
+                            <div
+                                className={`${classes.inputSectionWrapStyles} ${classes.wrapStyles}`}
+                            >
+                                <Typography
+                                    component="h2"
+                                    className={classes.titleStyles}
+                                >
+                                    Parent Company
+                                </Typography>
+                                <Typography
+                                    component="h4"
+                                    className={classes.subTitleStyles}
+                                >
+                                    Enter the company that holds majority
+                                    ownership for this production.
+                                </Typography>
+                                <StyledSelect
+                                    creatable
```

Comment:
Please remove the search icon from the creatable Select to be consistent with the designs.
To do it you need to add an customSelectComponents object: 
`const customSelectComponents = {
  DropdownIndicator: null,
};`
and pass it to the StyledSelect as a prop:
`components={customSelectComponents}`

Reply:
done.